cmd/queue/main.go

```diff
@@ -279,7 +303,7 @@ func buildServer(ctx context.Context, env config, healthState *health.State, rp
 		maxIdleConns = env.ContainerConcurrency
 	}
 
-	target := net.JoinHostPort("127.0.0.1", env.UserPort)
```

Comment:
once vHive's tests pass, we should change the code to use `env.UserPort` instead of introducing a new variable.

Reply:
Sorry but this is not clear to me. Did you mean that `env.GuestAddr` should remain as-is and `env.UserPort` should be used in lieu of `env.GuestPort`?

Reply:
right, we did this change in a hurry so we basically duplicated a parameter. anyways, let us test it first as is